manifests/l/Lexicool/LexibarES/1.4f/Lexicool.LexibarES.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Lexicool.LexibarES
+PackageVersion: 1.4f

```

```suggestion
PackageVersion: 1.4f
AppsAndFeaturesEntries:
- DisplayVersion:
```
The installer doesn't write a version to registry.